internal/api/oidc/config.go

```diff
@@ -18,7 +18,7 @@ type Config struct {
 	// provider for CLI login.
 	CLIClientID string `envconfig:"OIDC_CLI_CLIENT_ID"`
 	// Scopes are the scopes to be requested during the authorization code flow.
-	Scopes []string
+	Scopes string `envconfig:"OIDC_SCOPES"`
```

Comment:
We don't need to change this to a string. envconfig can unpack a comma-delimited string into a slice. Your struct tags you are using here do not even need to change. It will just work.

Reply:
Oh... but per [this comment](https://github.com/akuity/kargo/pull/2315/files#r1688313737) maybe this ought to change to:

```suggestion
        DefaultScopes []string
	AdditionalScopes []string `envconfig:"OIDC_ADDITIONAL_SCOPES"`
```

Reply:
That's cool that it just figures that out. I've moved them back to `[]string`s and move the envconfig to be used for additional scopes only.